internal/trie/node/variants.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package node
+
+type variant struct {
+	bits byte
+	mask byte
+}
+
+// Node variants
+// See https://spec.polkadot.network/#defn-node-header
+var (
+	leafVariant = variant{ // leaf 01
+		bits: 0b0100_0000,
+		mask: 0b1100_0000,
+	}
+	branchVariant = variant{ // branch 10
+		bits: 0b1000_0000,
+		mask: 0b1100_0000,
+	}
+	branchWithValueVariant = variant{ // branch 11
+		bits: 0b1100_0000,
```

Comment:
is this correct? Aren't the bits supposed to match this?
![Screenshot 2022-06-29 at 3 12 44 PM](https://user-images.githubusercontent.com/1837045/176517383-e5fbdb96-bb80-46f1-9860-31466c166cfd.png)

Reply:
Yes it's coming in the next PR